k8s/prod/common/ethos/repl-docmosis-service.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: repl-docmosis-service
+  namespace: ethos
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: repl-docmosis-service
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ethos-repl-docmosis-backend
+    version: 0.1.4
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '512Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "162CD82F0363F36E81218AD218C3C19F862CD480"
```

Comment:
Is that right app insights key ?

Reply:
Yes, I got it from the prod secrets. At some point it should be removed and created automatically as I don't think we are using App Insights properly.